### DISCUSSION TITLE
a/tech/opkg-conf

### DISCUSSION THREAD

**Discussion by giscus** [snowykami/blog#58] [Announcements]
# a/tech/opkg-conf

OPKG 工具 (一个 ipkg 变种) 是一个用来从本地软件仓库或互联网软件仓库上下载并安装 OpenWrt 软件包的轻量型软件包管理器。

https://sfkm.me/a/tech/opkg-conf

**Comment by yy-gk** [comment]
非常感谢，刚刚使用opkg，遇到的问题一模一样。
